### DISCUSSION TITLE
ADD AND UPGRADE TO MAKE BACKPACKS TO PLACE TORCHES FROM THEIR INVENTORY IN DARK PLACES.

### DISCUSSION THREAD

**Discussion by xelotex** [P3pp3rF1y/SophisticatedBackpacks#186] [Feature / Idea Archive]
broooo this would come very handy, maybe you could add a configurable area/reach level and light level for an advanced version of the upgrade or something like that.

**Comment by P3pp3rF1y** [comment]
First of all, all caps is considered just rude as it means you meant to scream at me!?

And this is beyond what I would like backpack to do, it definitely shouldn't be interacting with the outside world and physically placing blocks in the world. Also it's a functionality that Reliquary already has so why don't you try to play with that mod as well.

**Reply by xelotex** [reply]
Sorry didn't meant to be rude or something xd, i really appreciate your work and the last thing i want to do is being disrespectful against you, i was just trying to make the letters to look clear, but anyway, the reader can interpret text messages in many ways, and that's not the writer's fault xd.

ok fine don't worry if you don't want to add it, Reliquary... yeah i know, but i just need that "automatic torch placer" and nothing more from that mod, so having all the other features is a waste of memory, and that relic takes torches from the player's inventory and consumes torches REALLY fast, and my inventory is already too full to add two stacks of torches to it every 5 minutes, so nah, and imagine how cool it would be to have 2 stacks of 512 torches that only use 2 slots in the backpack, and by the way, making the backpacks to pickup items from the outside doesn't make them interact with the outside?.

**Reply by P3pp3rF1y** [reply]
That's where you can use sojourner's staff or void tear to store way more torches for you.

And when I say interacting with stuff outside I mean interacting with blocks as torches require block placement, interaction with entities sounds ok to me to a certain extent like magnet or pickup picking these up or feeding upgrade feeding players in very close proximity.
